### DISCUSSION TITLE
What is the long term goal for this project?

### DISCUSSION THREAD

**Discussion by Balssh** [prasanthrangan/hyprdots#196] [General]
First of all, many thanks to OP for this awesome project!
Now I’m curious about what’s the long term goal of this project: does OP want to remain something more personal to him, or to make it more configurable as a sort of installer (something akin to ArchTitus, but limited to GUI setup)?

**Comment by prasanthrangan** [comment]
Hi, thank you for your support!

Tbh, I just wanted to backup my personal configs and script to restore everything back in case I nuked my os drive.
As `archinstall` already works just fine for minimal install,  I made a script to install hyprland and essential required packages and restore my dots!

Regarding installation script, the goal is to
1. keep it simple
2. modular
3. re-runnable
4. replicatable for other systems(to enable others to fork and modify it for their use case)

Regarding the dots, the goal is to
1. consistent theming for _almost_ all apps
2. user friendly (documentation to explain default configs)
3. minimal look
4. modular
5. replicatable dots (to enable others to fork and modify it for their use case)

Anyways, this is just my hobby to mess around during my free time, whenever I have an idea/improvement...
But please let me know your thoughts/ideas/suggestions/improvements... feedbacks are always welcome :)
